adapta/security/clients/aws/_aws_client.py

```diff
@@ -33,8 +34,8 @@ class AwsClient(AuthenticationClient):
     """
 
     def __init__(self, aws_credentials: Optional[AccessKeyCredentials] = None):
-        self._session = None
-        self._credentials = aws_credentials or EnvironmentAwsCredentials()
+        self._session = boto3._get_default_session()
```

Comment:
calling `_xx()` methods on thrid-party libraries should be used with caution. Is there a solid reason for doing this?

Reply:
You're totally right and in the end it is not needed, I thought it was with the testing I did. 
I've removed it in [11247c6](https://github.com/SneaksAndData/adapta/pull/437/commits/11247c610a0a01c7dc2a7e504107bd99fcfda553)